### PR TITLE
Fix select prompt not clearing model property

### PR DESCRIPTION
**Pull request by williamhector**

### CODE REVIEW

File: addon/components/em-select.js [hunk 1]
```diff
@@ -99,7 +99,7 @@ export default Component.extend(InputComponentMixin, {
         if(selectedIndex !== 0){
           selectedIndex--;
         } else {
-          if (this.get('property')) {
+          if (this.get(this.get('property'))) {
```
Comment (spruce) [inline addon/components/em-select.js]: Have you tried this change? Because I think it should check on model if `this.get('property')` is set.

Comment (williamhector) [reply addon/components/em-select.js]: Sorry, I was trying to patch the issue that I just realised was fixed in 570dffd6f59f6bf7f290e700e445a58af422fea6.
Before that commit, it was `if (model.get('property')) {` and I fixed it locally by changing it to `if (model.get(this.get('property'))) {` thinking that was the intended check.
Then I didn't notice I was applying my fix to 570dffd6f59f6bf7f290e700e445a58af422fea6 which was fixed better anyway.

Comment (williamhector) [reply addon/components/em-select.js]: I have pointed my project to the master branch until that commit is released.

Comment (spruce) [reply addon/components/em-select.js]: Okay. Thanks for making this PR anyway. Gonna release that version soon. Hopefully with the other PR merged.

**Pull request closed without merging**